include/nudb/nudb.hpp

```diff
@@ -21,7 +20,8 @@
 #include <nudb/verify.hpp>
 #include <nudb/version.hpp>
 #include <nudb/visit.hpp>
-#include <nudb/win32_file.hpp>
```

Comment:
what happens if include the above on a posix or mac os platform?
>
> posix_file and win32_file are public interfaces depending on the macros, removing these includes will make one or both unavailable even if the user wants them available by setting the macros.
#include <nudb/native_file.hpp>

will include the one that will work in his environment.  It will create a portable program.  That's what I think users expect and what you intend.